### DISCUSSION TITLE
Annotations support

### DISCUSSION THREAD

**Discussion by seshrs** [eecs485staff/primer-spec#91] [Ideas]
Students probably use apps like OneNote, etc. for adding highlights, markup and notes. That said, I wonder whether students might find lightweight annotations on the page useful. Perhaps _highlighting_ segments of text, or adding _notes_ to segments of text.

At the same time, I'm concerned about the complexity this might add, both to the code and to the user experience. More investigation is needed to determine if such a feature makes sense for Primer Spec. (Perhaps it makes more sense as a standalone Chrome extension. EECS 441 project idea... 😉 )

**Comment by awdeorio** [comment]
I agree that this seems like a great idea for a separate code base.  It's not intrinsically related to primer-spec.
